multiversx_sdk_core/transaction.py

```diff
@@ -42,7 +42,7 @@ def __init__(self,
                  signature: Optional[bytes] = None,
                  guardian_signature: Optional[bytes] = None
                  ) -> None:
-        self.chainID = chain_id
+        self.chain_id = chain_id
```

Comment:
In theory, this is a breaking change. However, since users theoretically pass the `chain_id` in the constructor, instead of setting it via `transaction.chain_id`, plus that fact that `v0.7.0` was already quite breaking - it's OK to have this as `v0.7.1` :+1: